modules/theme/src/definitions/views/admin-portal.less

```diff
@@ -268,4 +239,134 @@
   height: 1px;
 }
 
+.wrap {
+	cursor: pointer;
+	float: left;
+	margin-left: -3px;
+	margin-right: 5px;
+}
+.arrow {
```

Comment:
`.arrow` class in the root level apply for all the `.arrow` components. Is this intentional ?